backend/src/main/java/ca/bc/gov/app/BootApplication.java

```diff
@@ -1,12 +1,19 @@
 package ca.bc.gov.app;
 
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Info;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@OpenAPIDefinition(info = @Info(
+    title = "${info.app.name}",
+    version = "${info.app.version}",
+    description = "${info.app.description}")
+)
 public class BootApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(BootApplication.class, args);
-	}
+  public static void main(String[] args) {
```

Comment:
@paulushcgcj it would be good to use the same formatter. I was using the default one of Eclipse.

Reply:
I'm using the google cheks on IntelliJ, it uses the google formatter

Reply:
Sounds good. I will use that one then.